connectivity/check/logging.go

```diff
@@ -329,3 +344,7 @@ func (a *Action) Fatalf(format string, s ...interface{}) {
 	a.fail()
 	a.test.Fatalf(format, s...)
 }
+
+func timestamp() string {
+	return fmt.Sprintf("[%s]", time.Now().Format(time.RFC3339))
```

Comment:
Nit: I'd add an additional space after the timestamp

```suggestion
	return fmt.Sprintf("[%s] ", time.Now().Format(time.RFC3339))
```